concent_api/core/tests/test_payments_bankster.py

```diff
@@ -259,23 +259,22 @@ def test_that_settle_overdue_acceptances_should_return_empty_claim_deposit_info_
                 self._get_list_of_force_transactions(),
             ]
         ) as get_list_of_payments_mock:
-            requestors_claim_payment_info = settle_overdue_acceptances(
+            claim_against_requestor = settle_overdue_acceptances(
                 requestor_ethereum_address=task_to_compute.requestor_ethereum_address,
                 provider_ethereum_address=task_to_compute.provider_ethereum_address,
                 acceptances=subtask_results_accepted_list,
+                requestor_public_key=hex_to_bytes_convert(task_to_compute.requestor_public_key),
             )
 
         get_list_of_payments_mock.assert_called()
 
-        self.assertIsNone(requestors_claim_payment_info.tx_hash)
-        self.assertIsNone(requestors_claim_payment_info.payment_ts)
-        self.assertEqual(requestors_claim_payment_info.amount_paid, 0)
-        self.assertEqual(requestors_claim_payment_info.amount_pending, 0)
+        self.assertEqual(
+            claim_against_requestor, 'timestamp_error')
 
     @override_settings(
         PAYMENT_DUE_TIME=10
     )
-    def test_that_settle_overdue_acceptances_should_return_claim_deposit_info_with_amount_pending_if_requestor_deposit_value_is_zero(self):
+    def test_that_settle_overdue_acceptances_should_return_claim_deposit_with_amount_pending_if_requestor_deposit_value_is_zero(self):
```

Comment:
you are not checking the `amount` field of the returned `DepositClaim` object, so either test name is wrong or some assertions are missing

Reply:
:+1: done